test/vonmisesfisher.jl

```diff
@@ -6,8 +6,10 @@ using LinearAlgebra, Test
 using SpecialFunctions
 
 vmfCp(p::Int, κ::Real) = (κ ^ (p/2 - 1)) / ((2π)^(p/2) * besseli(p/2-1, κ))
+logvmfCp(p::Int, κ::Real) = 1/2 * ((p - 2) * log(κ) - 2 * (log(besselix(p/2 - 1, κ)) +  κ) + p * (-log(2π)))
 
 safe_vmfpdf(μ::Vector, κ::Real, x::Vector) = vmfCp(length(μ), κ) * exp(κ * dot(μ, x))
```

Comment:
Can this be removed?

```suggestion
```

Reply:
Now it's just used to test the log-variant...

Reply:
Then let's remove it.

Reply:
sure!